tsdate/core.py

```diff
@@ -1054,15 +1060,17 @@ def get_dates(
     ignore_oldest_root=False,
     progress=False,
     cache_inside=False,
-    probability_space=base.LOG
+    probability_space=base.LOG,
 ):
     """
     Infer dates for the nodes in a tree sequence, returning an array of inferred dates
-    for nodes, plus other variables such as the distribution of posterior probabilities
+    for nodes, plus other variables such as the posteriors object
     etc. Parameters are identical to the date() method, which calls this method, then
     injects the resulting date estimates into the tree sequence
 
-    :return: tuple(mn_post, posterior, timepoints, eps, nodes_to_date)
+    :return: a tuple of (mn_post, posteriors, timepoints, eps, nodes_to_date).
+        If the "inside_outside" method is used, posteriors will contain the posterior
+        probabilities, else the returned variable will be ``None``.
```

Comment:
```suggestion
        probabilities for each node in each time slice, else the returned variable will be ``None``.
```

Reply:
Thanks a lot. I had to edit a rebase, so I'll make those changes separately.